web/components/ConferenceHeader/ConferenceHeader.tsx

```diff
@@ -0,0 +1,44 @@
+import { formatDateWithTime } from '../../util/date';
+import logo from '../../images/logo.svg';
+import styles from './ConferenceHeader.module.css';
+
+interface ConferenceHeaderProps {
+  name?: string;
+  startDate?: string;
+  endDate?: string;
+  description?: string;
+}
+
+export const ConferenceHeader = ({
+  name,
+  startDate,
+  endDate,
+  description,
+}: ConferenceHeaderProps) => {
+  return (
+    <div className={styles.root}>
+      <h1>
+        {/* eslint-disable-next-line @next/next/no-img-element */}
+        <img
```

Comment:
Two questions, mainly out of curiosity:
1. Why don't we use `next/image` here?
2. Since this content is very much "above the fold", should we consider inlining this SVG? (Inlining as a base64-encoded string in the HTML, that is. Not the "componentization" done in www-sanity-io which led to SVGs being parsed as JS)

Reply:
1. In short, because it's an SVG.

    In long: To me, next/image adds quite a bit of complexity when it comes to styling. E.g. it seems to be unable to size an image based on the height, so you have to hack around it by setting width to desired height * aspect ratio. There was a clipping bug on sanity.io related to this.
    
    Looking at its [main selling points](https://nextjs.org/docs/basic-features/image-optimization), they seem to be
    - "Always serve correctly sized image for each device": irrelevant for SVG as long as it's not transformed in some way to lose detail on smaller viewports. Here, it seems to set `srcset` and serve the exact same image on a different URL, which is pointless at best and wasteful at worst
    - "Prevent Cumulative Layout Shift automatically": we can just set the `width` and `height` attributes, that's a simpler way of accomplishing the same thing nowadays
    - "Images are only loaded when they enter the viewport": irrelevant here. Might still be useful in general, though it would be simpler to use `loading="lazy"` once Safari < 15.4 usage is sufficiently low
    - "with optional blur-up placeholders": irrelevant for SVG. Here, it seems to add an extra `<img>` with a completely blank image that seems like a waste to decode and render
    - "On-demand image resizing": same as first bullet point

2. Perhaps, but it's not clear to me if it's necessarily better or how much difference it would make. Filed a GitHub issue on it. (Using a base64-encoding sounds unnecessary, though; in this particular case it looks like it would add about 35% to the size. But it's possible to have `data:` URIs without base64.)

Reply:
Great, thanks for the in-depth explanation!